common/config.js

```diff
@@ -0,0 +1,72 @@
+require('dotenv').config()
+
+const production = process.env.NODE_ENV === 'production'
+
+function get(name, fallback, options = {}) {
+  if (process.env[name]) {
+    return process.env[name]
+  }
+  if (fallback !== undefined && (!production || !options.requireInProduction)) {
+    return fallback
+  }
+  throw new Error(`Missing env var ${name}`)
+}
+
+module.exports = {
+  sessionSecret: get('SESSION_SECRET', 'app-insecure-default-session', { requireInProduction: true }),
+  apis: {
+    oauth2: {
+      url: get('NOMIS_AUTH_URL', 'http://localhost:9090/auth', true),
+      externalUrl: get('NOMIS_AUTH_EXTERNAL_URL', get('NOMIS_AUTH_URL', 'http://localhost:9090/auth'), true),
+      timeout: {
+        response: get('AUTH_ENDPOINT_TIMEOUT_RESPONSE', 10000, true),
+        deadline: get('AUTH_ENDPOINT_TIMEOUT_DEADLINE', 10000, true),
+      },
+      agent: {
+        maxSockets: 100,
+        maxFreeSockets: 10,
+        freeSocketTimeout: 30000,
+      },
+      apiClientId: get('API_CLIENT_ID', 'use-of-force-client', true),
```

Comment:
do you have a sentence planner client id?

Reply:
Done.